cheroot/connections.py

```diff
@@ -5,7 +5,10 @@
 
 import io
 import os
-import select
+try:
+    import selectors
+except ImportError:
+    import selectors2 as selectors
```

Comment:
let's move this under `_compat` and just do
```python
from ._compat import selectors
```
in the local import section below